product_docs/docs/pem/9/monitoring_performance/notifications.mdx

```diff
@@ -265,3 +265,33 @@ You can use the following command to confirm that Nagios is properly configured:
 After confirming that Nagios is configured correctly, restart the Nagios service:
 
 `/usr/local/nagios/bin/nagios -d /usr/local/nagios/etc/nagios.cfg`
+
+## Customizing email templates
+
+Postgres Enterprise Manager monitors your system for conditions that require user attention and sends email notifications. Use the `Email Templates` to customize the email subject and the payload that the server sends if the current values deviate from the threshold values specified by an alert.
+
+These are the default email templates that are customizable:
+
+- Alert detected
+- Alert level increased
+- Alert level decreased
+- Alert cleared
+- Alert reminder
+- Job success
+- Job failure
+- Job cancellation
+- Job step
+- Job step (Database server)
+
+If any of the default alert template is customized, then a green tick mark is displayed against that template in the `Custom Template?` column.
```

Comment:
Not sure what you mean by "against"? Do you mean "next to"? 

Should this say "alert templates are" (plural)?

Reply:
changed